src/main/resources/templates/index.html

```diff
@@ -53,9 +53,9 @@
 
 <script>
     function search() {
-        let input = document.getElementById('filter').value;
+        var input = document.getElementById('filter').value;
         input = input.toLowerCase();
-        let x = document.getElementsByClassName('pokemon');
+        var x = document.getElementsByClassName('pokemon');
 
         for (i = 0; i < x.length; i++) {
             if (!x[i].innerHTML.toLocaleLowerCase().includes(input)) {
```

Comment:
in place of this if statement you could clean this up with the following 1 line ternary statement. 
x[i].innerHTML.toLocaleLowerCase().includes(input) ? x[i].style.display = "" : x[i].style.display = "none"